x-pack/plugins/security_solution/public/cloud_security_posture/components/csp_details/misconfiguration_findings_details_table.tsx

```diff
@@ -135,9 +134,9 @@ export const MisconfigurationFindingsDetailsTable = memo(
         width: '5%',
         render: (rule: CspBenchmarkRuleMetadata, finding: MisconfigurationFindingDetailFields) => (
           <EuiLink
-            onClick={() => {
-              navToFindingsByRuleAndResourceId(rule?.id, finding?.resource?.id);
-            }}
+            href={getFindingsPageUrlFilteredByRuleAndResourceId(rule?.id, finding?.resource?.id)}
```

Comment:
instead of implementing our own navigation, can we reuse existing components? Can you check `SecuritySolutionLinkAnchor` use in `x-pack/plugins/security_solution/public/flyout/entity_details/host_right/header.tsx` for example?